src/pages/guides/auth0.mdx

```diff
@@ -110,6 +110,15 @@ When integrating Web3Auth with Auth0 RWA, we need an additional backend server t
 - You will require `domain` and `clientId` of the newly created application which you can copy from [Auth0 Console](https://manage.auth0.com/).
   ![Domain and Client ID from Auth0 Dashboard](/contents/guides/auth0/auth0-guide-clientid.png)
 
+  ##### Email Passwordless
+
+  :::info
+
+  If you're to use Email Passwordless (Magic Link / Code) or SMS, please visit https://auth0.com/docs/authenticate/passwordless to setup the Lock
+  Screen of Auth0 from your Auth0 Dashboard.
```

Comment:
What's this?

Reply:
Enabling Auth0 Lock Screen(Modal) to use Email Passwordless and SMS.

Reply:
Oh right, sorry for missing this earlier.